packages/fx-core/src/question/create.ts

```diff
@@ -864,17 +864,18 @@ export class CapabilityOptions {
   // customize GPT
   static customizeGptBasic(): OptionItem {
     return {
-      id: "customize-gpt-basic",
-      label: "Basic GPT",
-      detail: "A simple GPT skeleton you can author without any plugin",
+      id: "basic-declarative-copilot",
+      label: "Basic Declarative Copilot",
```

Comment:
Please remember to do localization.

Reply:
sure! going to do that later. Currently this option is for CLI only.